sklearn/ensemble/_forest.py

```diff
@@ -2014,10 +2019,18 @@ class ExtraTreesRegressor(ForestRegressor):
     .. [1] P. Geurts, D. Ernst., and L. Wehenkel, "Extremely randomized trees",
            Machine Learning, 63(1), 3-42, 2006.
 
-    See Also
+    Examples
     --------
-    sklearn.tree.ExtraTreeRegressor: Base estimator for this ensemble.
-    RandomForestRegressor: Ensemble regressor using trees with optimal splits.
+    >>> from sklearn.datasets import load_boston
```

Comment:
Please do not use `load_boston` in examples. We are trying to get rid of it (see #16155). Maybe you can use `load_diabetes` instead?